access_med_utils/CMORise.py

```diff
@@ -21,7 +18,7 @@
 sys.path.append('/g/data/kj13/users/yz9299/app4/APP4/subroutines')
 sys.path.append('./')
 os.environ['ANCILLARY_FILES'] = '/g/data/p66/CMIP6/APP_ancils'
```

Comment:
@rhaegar325 We should avoid this as much as possible.

Reply:
I've confirmed there is an app_functions.py in the `kj13` directory shown above, and it's identical to the one that's in the repo under `access_med_utils`. Ergo, changing the import to point to the correct one within the code base is valid, and we should be able to remove at least the `sys.path.append('/g/data/kj13...')`.

Reply:
@rhaegar I have merged #9 but I realised that some of the issues were raised here. It would be good to keep contributions from everyone next time. Thanks anyway